sidebars.yaml

```diff
@@ -382,7 +384,7 @@ oldDocs:
             label: Fast Checkout Settings
           - page: developer-portal/woocommerce-migrate-plugin-directory.md
             label: Migrate to Plugin Directory
-  - group: I am on a custom system
```

Comment:
it looks like you're updating the old docs -- can you update the new ones? This sidebar content will be deleted once we launch the new docs.